graylog2-web-interface/src/views/components/widgets/Widget.test.tsx

```diff
@@ -376,33 +376,4 @@ describe('<Widget />', () => {
 
     expect(updateWidget).not.toHaveBeenCalledWith('widgetId', { config: { foo: 42 }, id: 'widgetId', type: 'dummy' });
   });
-
```

Comment:
I'm not sure why we are removing this test.

Reply:
I added a separate issue for that: https://github.com/Graylog2/graylog2-server/issues/21529